js/ui/grid_core/ui.grid_core.validating.js

```diff
@@ -718,7 +895,7 @@ module.exports = {
                             .addClass(INVALID_MESSAGE_CLASS)
                             .addClass(INVALID_MESSAGE_ALWAYS_CLASS)
                             .addClass(this.addWidgetPrefix(WIDGET_INVALID_MESSAGE_CLASS))
-                            .text(message)
+                            .html(message)
```

Comment:
Why? BC? Security?

Reply:
It's is required for displaying several error messages in the tooltip. There should not be problems with security, because I'll encode messages.